Jenkinsfile

```diff
@@ -68,10 +68,26 @@ pipeline {
             }
         }
 
+        stage('Run ORT scanner') {
+            steps {
+                sh 'rm -fr project/ort/scanner'
+                sh 'docker/run.sh "-v ${WORKSPACE}/project:/project" --info scan -a /project/ort/analyzer/analyzer-result.yml -o /project/ort/scanner'
+            }
+
+            post {
+                always {
+                    archiveArtifacts(
+                        artifacts: 'project/ort/scanner/*',
```

Comment:
This only archives files in this directory, but not subdirectories, correct? Otherwise it would archive downloaded source code and native scan results which might not be intended.

Reply:
Correct, it only matches contained files.